### DISCUSSION TITLE
Keybinds for TooManyEmotes?

### DISCUSSION THREAD

**Discussion by OurSapphireStar** [DaXcess/LCVR#178] [Q&A] [answered]
I've noticed that official support has been added for TooManyEmotes. I'm playing on Quest 2 and was wondering if it there is a way to set  keybinds somewhere for it or if I would have to make a custom profile using Unity?

**Comment by DaXcess** [comment, marked as answer]
The support only meant that the mods no longer interfere with each other, but in VR, none of the emotes mods will ever be able to make yourself emote, since the movement of the hands is forcibly tied to your own hands.
